ml_inference_offloading/src/androidTest/java/ai/nnstreamer/ml/inference/offloading/LocalDatabaseTest.kt

```diff
@@ -62,4 +66,46 @@ class LocalDatabaseTest {
             }
         }
     }
+
+    @Test
+    @Throws(Exception::class)
+    fun foreignKey() = runTest {
+        val modelName = "test"
+        val model = Model(
+            1,
+            modelName
+        )
+        modelDao.insert(model)
+
+        val srcCaps =
+            "other/tensors,num_tensors=1,format=static,dimensions=(string)3:224:224:1,types=uint8,framerate=0/1"
+        val sinkCaps =
+            "other/tensors,num_tensors=1,format=static,dimensions=(string)1001:1,types=uint8,framerate=0/1"
+        val properties = "framework=tensorflow-lite"
+        val pipelineConfig = PipelineConfig(
+            1,
+            srcCaps,
+            sinkCaps,
+            properties
+        )
+        pipelineConfigDao.insert(pipelineConfig)
+
+        val pipeline = ai.nnstreamer.ml.inference.offloading.data.Pipeline(
```

Comment:
`Pipeline` for Kotlin data class name could cause confusion.
Would it be better to use another name?